meshio/med_io.py

```diff
@@ -51,21 +51,24 @@ def read(filename):
     number = pts_dataset.attrs["NBR"]
     points = pts_dataset[()].reshape(-1, number).T
 
-    # Cells
-    cells = {}
-    mai = mesh["MAI"]
-    for med_cell_type, med_cell_type_group in mai.items():
-        cell_type = med_to_meshio_type[med_cell_type]
-        cells[cell_type] = med_cell_type_group["NOD"][()].reshape(
-            num_nodes_per_cell[cell_type], -1).T - 1
-
     # Read nodal and cell data if they exist
     try:
         cha = f["CHA"]  # champs (fields) in french
         point_data, cell_data, field_data = _read_data(cha)
     except KeyError:
         point_data, cell_data, field_data = {}, {}, {}
 
+    # Cells
+    cells = {}
+    mai = mesh["MAI"]
+    for med_cell_type, med_cell_type_group in mai.items():
+        cell_type = med_to_meshio_type[med_cell_type]
+        families = med_cell_type_group["FAM"][()]
```

Comment:
`"FAM"` may not exist so this may cause an error.

Reply:
Oh.  I was going on the example posted to #347 by @RemiTheWarrior.  I'll retract this proposal; I don't understand MED well enough.

I do recommend 776ab4a though, I think.